### ISSUE TITLE
Is it possible to install a GUI in TermuxAlpine?

### ISSUE THREAD

**Issue by MrJinx123** [Hax4us/TermuxAlpine#12]
I have been trying the instructions on Alpine's WIKI trying to get a working GUI but can't seem to find the magic. Xorg, X11VNC and my preferred.. xRDP are not able to be started correctly. The simplest X11 export DISPLAY=0 or any variation commands fail. xRDP launches and logs in but can't reach a desktop (MATE) or gui app directly. 
It seems since rc-service is not started due to proot config some functions just won't start or nest correctly.
There are 2 other details I ran into with the package. 
1. I had to change TMPDIR to use the rootfs/tmp directory.. Openjdk8 was having heartburn with the termux env path. <-under the current proot config, I don't think it can reach it.. 
2. To use xRDP you have to add a user. That process works fine but login fails after being successful due to bash config from termux env. responds with  bad shell and kicks you back to root account bash config..
I would love to use this for the few linux apps there are not android ports for. I don't really care about a full linux desktop on Android x86_64. (Running PrimeOS)For now , I have Android Studio, Serviio Media Server running fine on Proot Ubuntu.. AQEMU qemu front end running well on termux directly..VirtualBox is the only other target left I don't have a good android port for already..
i really like the footprint (size) of Alpine if the x11 support was just a little more do-able..
  Thanks!!

**Comment by iL3D** [comment]
Yes it is very possible. The script [setup-termux-alpine-gui.sh](https://gist.github.com/iL3D/10154b03c8f9cc105cc3d8162596b273) demonstrates one way to run X apps and Java 8 apps under TermuxAlpine. Hope that helps.

**Comment by apandada1** [comment]
An alternative would be to use the xfce setup file by Andronix.
https://raw.githubusercontent.com/AndronixApp/AndronixOrigin/master/Installer/Alpine/alpine-xfce.sh

This repo hosts a more recent version of Alpine compared to Andronix.
